parsec/api/protocol/realm.py

```diff
@@ -62,6 +63,15 @@ class RealmStatusRepSchema(BaseRepSchema):
 
 realm_status_serializer = CmdSerializer(RealmStatusReqSchema, RealmStatusRepSchema)
 
+class RealmStatsReqSchema(BaseReqSchema):
+    realm_id = fields.UUID(required=True)
+
+
+class RealmStatsRepSchema(BaseRepSchema):
+    data_size = fields.Integer(required=True)
+
+realm_stats_serializer = CmdSerializer(RealmStatsReqSchema, RealmStatsRepSchema)
```

Comment:
nitpicking: 2 `\n` between top-level definitions ;-p

black autoformater doesn't enforce this here given we are between a class definition and a variable declaration (see https://black.readthedocs.io/en/stable/the_black_code_style.html#empty-lines), but it's the style we use in most of the code